src/app/core/models/auth/auth.class.ts

```
@@ -0,0 +1,12 @@
+export class Auth {
+  private static readonly baseUrl = 'api/login';
+  private static readonly refreshUrl = 'api/refresh';
+
+  static get BaseUrl(): string {
+    return this.baseUrl;
+  }
+
+  static get RefreshUrl(): string {
+    return this.refreshUrl;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)